code/modules/modular_computers/file_system/programs/command/card.dm

```diff
@@ -151,7 +151,7 @@
 				else
 					var/contents = {"<h4>Crew Manifest</h4>
 									<br>
-									[SSrecords.get_manifest(1)]
+									[SSrecords.get_manifest(monochrome = 1)]
```

Comment:
doesn't this runtime since you removed SSrecords.get_manifest?

Reply:
It was not removed, it just moved from line 182 to [**193**](https://github.com/Aurorastation/Aurora.3/blob/1669f46772ae68b496fb1d9eebe36a58f044d839/code/controllers/subsystems/records.dm#L193-L240) and was somewhat modified. This is one of the parts I explicitly tested (I actually hope I tested all the cases but you can never be 100% sure), and the printed paper comes out more or less like previously.

Anyway, thank you for the thoroughness.